pjsip/include/pjsua2/types.hpp

```diff
@@ -52,6 +52,9 @@ typedef std::vector<std::string> StringVector;
 /** Array of integers */
 typedef std::vector<int> IntVector;
 
+/** Array of bytes */
+typedef std::vector<unsigned char> ByteVector;
```

Comment:
More like a note rather than a suggestion.

IMO bit number is quite urgent as our media components moslty work with 8 or 16 bit audio sample, so was about to propose replacing term `byte` to `octet`, as byte may be more/less than 8 bits (see [here](https://stackoverflow.com/questions/18576822/sizeof-and-when-a-byte-is-larger-than-8-bits#comment27335390_18576901)), while `octet` is always 8 bits. But not sure if it is still relevant on current hardwares.

Also, was considering `unsigned char` vs `pj_uint8_t` for a similar reason, but SWIG macros may not recognize `pj_uint8_t` and some media parts in our library seem to use `unsigned char` too.

Reply:
Noted. After experimenting with various data types, I now know how difficult it is to get the bit number correct, and each target language may behave differently and present different challenges.